docs/DeveloperGuide.md

```diff
@@ -1024,6 +1057,46 @@ MSS:
   * 4a1. QuickCache shows an error message.
 
     Use case resumes at step 3.
+    
+    
+**Use case: UC15 - Edit a flashcard**
+
+**Actor: User**
+**Preconditions: User has QuickCache open.**
+
+**MSS**
+
+1.  User requests to list flashcards
+2.  QuickCache shows a list of flashcards
+3.  User request to edit a specific flashcard of the list
+4.  QuickCache edit the flashcard
+5.  QuickCache updates flashcard save file (UC08)
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. The list is empty.
+
+    Use case ends.
+    
+* 3a. The given index in invalid.
+
+  * 3a1. QuickCache shows an error message.
+
+    Use case resumes at step 2.
+    
+* 3b. There is no change in the flashcard or the flashcard already exists.
```

Comment:
maybe its better to word it as "or the newly edited flashcard is the same as another flashcard" instead of

> or the flashcard already exists.

Reply:
donee